webpack.config.js

```diff
@@ -17,6 +17,7 @@ const config = {
   entry: {
     'background': './background/index.js',
     'popup/popup': './popup/popup.js',
```

Comment:
should we rename `popup.js` to `index.js` to be consistent with background and onboarding entries?

Reply:
yes, but it should be done in master branch